test/binary_search_tree_test.rb

```diff
@@ -88,4 +88,24 @@ def test_it_can_insert_a_twelfth_node
     assert_equal 3, result
   end
 
+  def test_include?
+    @tree.insert(61, "Bill & Ted's Excellent Adventure")
+
+    assert @tree.include?(61)
+  end
```

Comment:
True TDD. Obv. the required methods don't exist yet, but here's some ideas.

These comments even have nice code formatting options:

```ruby
def test_it_can_insert_a_twelfth_node
    @tree.insert(61, "Bill & Ted's Excellent Adventure")
    
    assert stuff
  end
```

Finally, you can @Shniks  someone and they'll get a notification to take a look.